packages/zosfiles/src/api/constants/ZosFiles.messages.ts

```diff
@@ -555,6 +547,14 @@ export const ZosFilesMessages: { [key: string]: IMessageDefinition } = {
         message: "Data set migraton requested."
```

Comment:
"migraton" -> "migration"

Reply:
Im going to add the "spell checker" extention to my vscode 😂 , good spot!